jekyll/_cci2/api-job-trigger.md

```diff
@@ -47,14 +47,12 @@ For a complete reference of the API,
 see the [CircleCI API Documentation]({{ site.baseurl }}/api/v1-reference/).
 
 **Note:**
-While you can **not** trigger [workflows]({{ site.baseurl }}/2.0/workflows/) with the CircleCI API,
-the `config.yml` that contains the job definition
-can still have a `workflows` section.
+It is possible to trigger [workflows]({{ site.baseurl }}/2.0/workflows/) with the CircleCI API, using a new endpoint, see the Trigger a Build with a Workflow section of the [CircleCI API Projects Documentation]({{ site.baseurl }}/api/v1-reference/#projects).
```

Comment:
In the other file it's called "Trigger a new Build by Project" - should either change here to reflect that or change there to reflect this.

Reply:
thanks, changed it here